jumpscale/packages/owncloud/package.py

```diff
@@ -0,0 +1,25 @@
+import toml
+from pathlib import Path
+
+PACKAGE_CONFIG_PATH = Path(__file__).parent.resolve() / 'package.toml'
+class owncloud:
+    def install(self, **kwargs):
+        domain = kwargs.get('domain')
+        letsencryptemail = kwargs.get('letsencryptemail')
+        if not all([domain, letsencryptemail]):
+            raise(f'usage: j.servers.threebot.default.packages.add(path="<package_path>", domain="<domain>", letsencryptemail="<letsencryptemail>")')
```

Comment:
I think we can have a proper error message in addition to this usage information?

Reply:
done, also it shows a warning now instead of raising an error.